examples/apps/colorapp/ecs/colorgateway-base-task-def.json

```diff
@@ -0,0 +1,112 @@
+{
+  "family": $NAME,
+  "proxyConfiguration":{
+      "type": "APPMESH",
+      "containerName": "envoy",
+      "properties": [
+          {
+              "name": "IgnoredUID",
+              "value": "1337"
+          },
+          {
+              "name": "ProxyIngressPort",
+              "value": "15000"
+          },
+          {
+              "name": "ProxyEgressPort",
+              "value": "15001"
+          },
+          {
+              "name": "AppPorts",
+              "value": "9080"
+          },
+          {
+              "name": "EgressIgnoredIPs",
+              "value": "169.254.170.2,169.254.169.254"
+          }
+      ]
+  },
+  "containerDefinitions": [
+    {
+      "name": "app",
+      "image": $APP_IMAGE,
+      "portMappings": [
+        {
+          "containerPort": 9080,
+          "hostPort": 9080,
+          "protocol": "tcp"
+        }
+      ],
+      "environment": [
+        {
+          "name": "SERVER_PORT",
+          "value": "9080"
+        },
+        {
+          "name": "COLOR_TELLER_ENDPOINT",
+          "value": $COLOR_TELLER_ENDPOINT
+        },
+        {
+          "name": "TCP_ECHO_ENDPOINT",
+          "value": $TCP_ECHO_ENDPOINT
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": $ECS_SERVICE_LOG_GROUP,
+          "awslogs-region": $AWS_REGION,
+          "awslogs-stream-prefix": $AWS_LOG_STREAM_PREFIX_APP
+        }
+      },  
+      "essential": true,
+      "dependsOn": [
+        {
+          "containerName": "envoy",
+          "condition": "HEALTHY"
+        }
+      ]
+    },
+    {
+      "name": "envoy",
+      "image": $ENVOY_IMAGE,
+      "user": "1337",
+      "essential": true,
+      "environment": [
+        {
+          "name": "APPMESH_VIRTUAL_NODE_NAME",
+          "value": $VIRTUAL_NODE
+        },
+        {
+          "name": "ENVOY_LOG_LEVEL",
+          "value": $ENVOY_LOG_LEVEL
+        },
+        {
+          "name": "APPMESH_XDS_ENDPOINT",
+          "value": $APPMESH_XDS_ENDPOINT
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": $ECS_SERVICE_LOG_GROUP,
+          "awslogs-region": $AWS_REGION,
+          "awslogs-stream-prefix": $AWS_LOG_STREAM_PREFIX_ENVOY
+        }
+      },
+      "healthCheck": {
+        "command": [
+          "CMD-SHELL",
+          "echo hello"
```

Comment:
this should probably be something like 

`"curl -f http://localhost:9901/server_info || exit 1"`

Reply:
Updated this. Thanks :)